PlayCover/Utils/ViewExtenstions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ViewExtenstions.swift
+//  PlayCover
+//
+//  Created by Amir Mohammadi on 9/26/1401 AP.
+//
+
+import SwiftUI
+
+struct BadgeTextStyle: ViewModifier {
+    func body(content: Content) -> some View {
+        if #available(macOS 13.0, *) {
+            content
+                .textCase(.uppercase)
+                .font(.subheadline)
+                .bold()
+                .foregroundColor(Color(nsColor: .tertiaryLabelColor))
+        }
+    }
+}
+
+struct VerticalSpacer: View {
```

Comment:
This struct should be in the same swift file as the one where it is used. It is a nice shorthand used in one view, not a global utility.

Reply:
In first place I didn't separated files and did it to avoid SwiftLint errors, I'll change 👍